test-cases/RMLTC0015b-MySQL/mapping.ttl

```diff
@@ -26,16 +26,16 @@
 <http://example.com/base/TriplesMap2> a rml:TriplesMap;
   rml:logicalSource [
       rml:iterator """
-            SELECT \"Code\", \"Name\", \"Lan\"
-            FROM \"Country\"
-            WHERE \"Lan\" = 'ES';
-           """;
+        SELECT Code, Name, Lan
```

Comment:
Why changing this completely?

Reply:
because it is invalid. The escaping is unnecessary in a multiline string

Reply:
so this leads to an error, but not the expected error in the test case